test/test_tensordict.py

```diff
@@ -1383,6 +1385,13 @@ def test_getitem_string(self, td_name, device):
         td = getattr(self, td_name)(device)
         assert isinstance(td["a"], (MemmapTensor, torch.Tensor))
 
+    def test_getitem_range(self, td_name, device):
+        torch.manual_seed(1)
+        td = getattr(self, td_name)(device)
+        assert_allclose_td(td[range(2)], td[[0, 1]])
+        assert_allclose_td(td[range(1), range(1)], td[[0], [0]])
+        assert_allclose_td(td[:, range(2)], td[:, [0, 1]])
```

Comment:
let's add Ellipsis here

Reply:
Added a test for indexing with `Ellipsis`.